crypto/vrf.go

```diff
@@ -0,0 +1,26 @@
+package crypto
+
+import (
+	"bytes"
+	"errors"
+)
+
+type VRFSigner struct {
+	privateKey []byte
```

Comment:
should it be fixed size maybe?

Reply:
this is a sort of stub, when we implement the signer we can change this